Eastern-Research-Group/mywaterway
# Feature/state page aquatic life tab

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3183005

## Main Changes:
* Added a fifth tab, Aquatic Life, to State Page
* 5 tabs are not feasible to fit on a 400px screen so on small devices they now are full width and stacked. 

## Steps To Test:
1. Navigate to State Water Quality Overview page, try several states and compare the Eating Fish and Aquatic Life tabs. The Uses for each should only be related to consumption of fish or the status of wildlife. 
2. Test mobile responsiveness for Topic tabs on State page Water Quality Overview.
